### DISCUSSION TITLE
Disable-ScheduledTask is not recognized

### DISCUSSION THREAD

**Discussion by tocguy** [ChrisTitusTech/winutil#2771] [Q&A]
PowerShell 7 in Admin mode on Win 10 Pro 22H2 in Admin account.

Many of these Unhandled Exception errors being thrown by the Utility.

WARNING: The term 'Disable-ScheduledTask' is not recognized as a name of a cmdlet, function, script file, or executable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.

**Comment by arielsil** [comment]
Drives me crazy on Win Server all versions 20348 ... 26304.   Yes, even those later than 25314.
